networkx/utils/decorators.py

```diff
@@ -911,10 +909,9 @@ def compile(self, f):
         return func
 
     def assemble(self, f):
-        """Collects the requisite info to compile the decorated version of f.
-
-        Note, this is recursive, and all nested argmap-decorators for a
-        specific decorated function will be flattened into a single function.
+        """Collects components of the source for the decorated function wrapping f.
+        If f is argmap-decorated, then we recursively assemble the stack of
+        decorators into a single flattened function.
```

Comment:
I think sphinx needs a blank line after the initial one-line description (used for documentation links *to* this method).  Hence the suggestion of a blank line.

The recursion occurs only for nested-argmap-decoration, correct?
```suggestion

        If f has multiple argmap decorators, we recursively assemble the stack of
        decorators into a single flattened function.
```